yb-voyager/cmd/importSchemaYugabyteDB.go

```diff
@@ -92,5 +92,8 @@ func applySchemaObjectFilterFlags(importObjectOrderList []string) []string {
 	} else {
 		finalImportObjectList = utils.SetDifference(importObjectOrderList, excludeObjectList)
 	}
+	if sourceDBType == "postgresql" && !slices.Contains(finalImportObjectList, "SCHEMA") { // Schema should be migrated by default.
```

Comment:
@rahulb-yb if the source schema is passed in the arg then this should be fine no? We can assume that the user is looking for that schema and just check if it exists or not in the target, and  create it if it doesn't. We know about the schema from the exported sql files? Does it make sense?

Reply:
For import schema, we simply replay the .sql files that have been exported, this is based on object type rather than one (or more) specific schema to import into. When the user does not include `schema` in the object list, we do not make any schemas available for importing the remainder of the schema to be consolidated into

This is the simplest approach I could think of which considers all scenarios, simply append the `schema` object type if it already doesn't exist, to the list of object types we are to import on the target

Reply:
ok. sounds good.